man/systemd-cat.xml

```diff
@@ -92,6 +92,18 @@
         see <option>--level-prefix=</option> below.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--stderr-priority=</option></term>
+
+        <listitem><para>Specifies the default priority level for
+        messages from the process's standard error output (stderr).
+        Usage of this option is the same as the
+        <option>--priority=</option> option, above, and both can be
+        used at once. When both are used, <option>--priority=</option>
+        will specify the default priority for standard output (stdout).
```

Comment:
key info is missing here I think: that if --stderr-priority= is *not* specified the stderr priority is the same as the stdout priority.

Reply:
hmm, i think an additional note mentioning that if the options are picked differing from each other messages to stdout and stderr lose their strict ordering properties. (because they are then dispatched over two different individually ordered channels instead of a single ordered one)

Reply:
Ah, yes, good point!  I've extended the manual description to cover those topics.